src/doctokenizer.l

```diff
@@ -57,8 +57,10 @@ static yy_size_t g_inputPos = 0;
 static const char *g_inputString;
 static QCString g_fileName;
 static bool g_insidePre;
+static bool g_insideQuot;
 static int g_sharpCount=0;
-static bool g_markdownSupport=TRUE;
+static bool g_markdownSupport = TRUE;
+static bool g_quotSupport = FALSE;
```

Comment:
I guess, both ``g_markdownSupport`` and ``g_quotSupport`` shouldn't be initialized here since they are initialized in  ``doctokenizerYYinit``.